odoo110_py37/files/odoo_cron_service.conf

```diff
@@ -0,0 +1,10 @@
+[program:odoo_cron]
+user=odoo
+command=/home/odoo/instance/odoo/odoo-bin -c /home/odoo/.openerp_serverrc --no-http --workers=0 --max-cron-threads=10 --pidfile=/tmp/odoo_cron.pid
+stdout_logfile=/var/log/supervisor/odoo_stdout.log
+stderr_logfile=/var/log/supervisor/odoo_stdout.log
```

Comment:
@moylop260 In the call with @nhomar suggested to use a different name for the log files, do we leave the same or change it?

Reply:
I just left the same but I just increase the rotate size to 1Gb in order to avoid rotating twice

But tell me if you prefer to use another log file

No problem, only if they will be read by graylog